Helper/Vault.php

```diff
@@ -147,12 +147,6 @@ private function getVaultPaymentToken($payment, array $additionalData): PaymentT
             );
 
             $paymentToken->setGatewayToken($additionalData[self::RECURRING_DETAIL_REFERENCE]);
-
-            if (strpos($additionalData[self::PAYMENT_METHOD], "paywithgoogle") !== false
-                && !empty($additionalData['paymentMethodVariant'])) {
-                $additionalData[self::PAYMENT_METHOD] = $additionalData['paymentMethodVariant'];
-                $paymentToken->setIsVisible(false);
```

Comment:
Removing this means that payments made with Google Pay will still be stored, just as not visible?
Google Pay supports recurring payments https://docs.adyen.com/payment-methods/google-pay/api-only#recurring-payments

Reply:
I tested and the google pay  token is already set as visible false